examples/asset.py

```diff
@@ -0,0 +1,88 @@
+from pyteal import *
+
+def approval_program():
+    on_creation = Seq([
+        Assert(Txn.application_args.length() == Int(1)),
+        App.globalPut(Bytes("total supply"), Btoi(Txn.application_args[0])),
+        App.globalPut(Bytes("reserve"), Btoi(Txn.application_args[0])),
+        App.localPut(Int(0), Bytes("admin"), Int(1)),
+        App.localPut(Int(0), Bytes("balance"), Int(0)),
+        Return(Int(1))
+    ])
+
+    is_admin = App.localGet(Int(0), Bytes("admin"))
+
+    on_closeout = Seq([
+        App.globalPut(
+            Bytes("reserve"),
+            App.globalGet(Bytes("reserve")) + App.localGet(Int(0), Bytes("balance"))
+        ),
+        Return(Int(1))
+    ])
+
+    register = Seq([
+        App.localPut(Int(0), Bytes("balance"), Int(0)),
+        Return(Int(1))
+    ])
+
+    # configure the admin status of the account Txn.accounts[0]
+    # sender must be admin
+    new_admin_status = Btoi(Txn.application_args[1])
+    set_admin = Seq([
+        Assert(And(is_admin, Txn.application_args.length() == Int(2))),
+        App.localPut(Int(1), Bytes("admin"), new_admin_status),
+        Return(Int(1))
```

Comment:
This might not be a bad place to add a small comment documenting why `is_admin` lives up in the `Assert` and not down here in the `Return`.